keps/sig-node/4800-cpumanager-split-uncorecache/README.md

```diff
@@ -0,0 +1,672 @@
+
+- prefer-align-cpus-by-uncorecache will be compatible with the default CPU allocation logic with future plans to be compatible with the options distribute-cpus-across-numa and distribute-cpus-across-cores.
+
+### User Stories (Optional)
+
+#### Story 1
+
+- As a HPC user, I want to extract the best performance possible and reduce latency, so that I can extract the most value when deploying applications in Kubernetes. I can use the static CPU Manager Policy option, but on split uncore cache processors, my application can experience latency when assigned CPUs across multiple uncore caches. In order to maximize performance, I want to minimize the distribution across uncore caches to minimize the latency. I want a feature I can enable so the CPU allocation logic inside the kubelet will automatically handle this for me for larger clusters for simple deployment and efficiency.
+
+#### Story 2
+
+- As a Networking/Telco Engineer, my application is latency sensitive, to which uncore cache misalignment is a contributing factor. I want my application to be statically assigned to CPUs that correspond to one single uncore cache so that I can get the maximum throughput from my application. Additionally, I have applications that are also static and of different CPU size requirements. The default Static CPU Manager Policy will result in my applications being assigned to two or more uncore caches. I have a large multi-node Kuberentes cluster. I want a cpu allocation logic to automatically assign CPUs for the best performance across the fewest amount of uncore cahces based on available CPUs without having to worry about what order/size I deploy my applications so I can ensure I get the best performance possible. 
+
+### Notes/Constraints/Caveats (Optional)
+
+- The name UnCoreCache is directly derived from cAdvisor that is being used as a package. 
+- Using the flow in `CPUManagerPolicy{Alpha,Beta}Options` from [xref](https://github.com/kubernetes/kubernetes/blob/af879aebb1a866a2f0e45bb33c09a1cc8f7acc45/pkg/features/kube_features.go#L110C1-L117C83) which is used to avoid a proliferation of a feature gates. 
+
+### Risks and Mitigations
+
+- Risk: Enabling or disabling the feature might lead to unexpected behavior
+  - Mitigation: Feature is enabled by a static policy option flag
+- Risk: The new feature might interfere with existing functionality
+  - Mitigation: It does not change behavior of non-static policy and preserves the behavior of other static options
+- Risk: Inconsistent configuration could cause scheduling issues
+  - Mitigation: Failure will occur if mismatch between options occurs, preventing the Pod from being scheduled incorrectly or leading a non-optimal aligment
```

Comment:
Can we clarify where the failure occurs? As written it suggests: 
1. We prevent incompatible options from being specifie and fail on kubelet config validation. 
OR
2. We fail at runtime when trying to place a pod when incompatible options are specified which is _not_ desirable.

Reply:
We are suggesting the 2nd option that we would fail at runtime

Reply:
I went ahead and added a small change, does that clear it up a bit?